routing.go

```diff
@@ -0,0 +1,35 @@
+package httpapi
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+)
+
+type RoutingAPI HttpApi
+
+func (api *RoutingAPI) Get(ctx context.Context, k string) (out []byte, err error) {
+	resp, err := api.core().Request("dht/findpeer", k).Send(ctx)
```

Comment:
There is `routing/get`: https://docs.ipfs.tech/reference/kubo/rpc/#api-v0-routing-get

Reply:
See:
- https://github.com/ipfs/kubo/pull/9628
- https://github.com/ipfs/kubo/pull/9628
- https://github.com/ipfs/interface-go-ipfs-core/pull/104
- https://github.com/ipfs/kubo/pull/9629